src/routes/product.route.js

```diff
@@ -19,4 +19,7 @@ productRoute.get("/recommended", ProductController.getRecommendedProducts);
 productRoute.post("/create",verifyVendor,uploadImages("productImage"),validate(productSchema),productExistAlready,ProductController.createProduct);
 productRoute.patch("/update/:id",verifyVendor,uploadImages("productImage"),IsProductExist,validate(updateSchema),ProductController.updateProduct);
 productRoute.delete("/delete/:id",verifyVendor,ProductController.deleteProduct);
+productRoute.get("/seller/getall",verifyVendor,ProductController.availableProductsInCollection);
```

Comment:
Same as here; Use hyphens to separate words in the endpoint names makes them more readable and understandable, especially when there are multiple words involved.